doc/bridge_packages/flame_forge2d/forge2d.md

```diff
@@ -63,6 +63,42 @@ useful if you want to add for example animations or other components on top of y
 The body that you create in `createBody` should be defined according to Flame's coordinate system,
 not according to the coordinate system of Forge2D (where the Y-axis is flipped).
 
+:exclamation: In Forge2D (you shouldn't add any bodies as children to other components.
+Forge2D doesn't have a concept of nested bodies.
+So bodies should live on the top level in the physics world.
+Consequently, below is **wrong**
+
+```dart
+class Bullet extends BodyComponent  {
+
+  @override
+  Future<void> onLoad() async {
+    ...
+    isBullet = true;
```

Comment:
I don't think you need to have this in this example to keep the example minimal and only about the concept that is explained here.